relay-profiling/src/cocoa.rs

```diff
@@ -121,11 +121,31 @@ struct CocoaProfile {
     version_name: String,
 }
 
+impl CocoaProfile {
+    fn filter_samples(&mut self) {
+        let mut sample_count_by_thread_id: HashMap<u64, u32> = HashMap::new();
+
+        for sample in self.sampled_profile.samples.iter() {
+            *sample_count_by_thread_id
+                .entry(sample.thread_id)
+                .or_default() += 1;
+        }
+
+        sample_count_by_thread_id.retain(|_, count| *count > 1);
```

Comment:
`1` is the only limit we'd want. Basically, we need at least 2 samples (list of frames executed at a given timestamp) to calculate a duration. So it's not going to change, I'll remove every thread with only 1 sample since I can't do anything interesting with this data.

Reply:
Perfect, thanks!